director/jobs/jobs.py

```diff
@@ -0,0 +1,132 @@
+from celery import Celery, signature
+from celery.result import AsyncResult
+from django.conf import settings
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from jobs.models import Job, JobMethod, JobStatus
+from projects.models import Project
+from users.models import User
+
+# Setup the Celery app
+# This is used to send and cancel jobs
+celery = Celery("director", broker=settings.BROKER_URL, backend="rpc://")
+celery.conf.update(
+    # By default Celery will keep on trying to connect to the broker forever
+    # This overrides that. Initially try again immediately, then add 0.5 seconds for each
+    # subsequent try (with a maximum of 3 seconds).
+    # See https://github.com/celery/celery/issues/4296
+    broker_transport_options={
+        "max_retries": 10,
+        "interval_start": 0,
+        "interval_step": 0.5,
+        "interval_max": 3,
+    },
+    # Needed to ensure STARTED state is emitted
+    task_track_started=True,
+)
+
+
+def send_job(queue: str, id: int, method: JobMethod, params: dict):
```

Comment:
`id` is a built in function in python and shouldn't be shadowed as a variable name.